js/npm/arangojs/Database.hx

```diff
@@ -59,6 +59,8 @@ implements npm.Package.RequireNamespace<"arangojs", "^4.3.0">
 	// Queries
 	// aqlQuery macro
 
+        @:overload(function<T>(query : { query: Either<String, Aqb>, ?bindVars : {} }, opts : {}, cb : ArangoCallback<Cursor<T>>) : Void {})
```

Comment:
then `Either<String, Aqb>` resolves to `Either<String, Either<String, { function toAQL() : String; }>>` ?
this all looks weird.
I can see it was already weird before your commit, but still...